### DISCUSSION TITLE
Export query to CSV without string delimiters

### DISCUSSION THREAD

**Discussion by lestevam** [dbeaver/dbeaver#13271] [Q&A] [answered]
#### System information: 
- Windows 10
- DBeaver version 21.1.2

#### Connection specification:
- SqlServer 2016
- Microsoft JDBC Driver for SQL Server
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
When we export a query to CSV and in CSV configuration we don't put delimiter characters, we see the result with quotes in the column.

#### Steps to reproduce, if exist:
![image](https://user-images.githubusercontent.com/917757/125822509-6bfa41b2-8f48-40cd-ba50-23918d687264.png)
 
![image](https://user-images.githubusercontent.com/917757/125822540-50b91bc7-5856-4cc1-867c-c4587e9ac9f9.png)

**Comment by uslss** [comment, marked as answer]
@lestevam 
seems you should check Quote never option in this case
![image](https://user-images.githubusercontent.com/31996417/126135290-7093d099-f0c7-4ba6-82ca-d0814844a172.png)

**Comment by lestevam** [comment]
HI @uslss, you are right!

I think that will be interesting when we selected the disabled in the "Quote always" the "Quote never" automatically change to "true" and any other configuration in the "Quote always" change the "Quote never" automatically to "false".

Thank you for your help!
